.github/workflows/tests.yml

```
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9]
```

I propose extending the list of covered Python versions. Also, let's use quotes to avoid floats (3.10 -> 3.1)
```suggestion
        python-version: ["3.8", "3.9", "3.10"]
```